### PR TITLE
Created the Home section 

### PR DESCRIPTION
**Pull request by Elijahdre**
# In this project, I ; 
- Created the head section of my portfolio project using HTML and CSS code.
- Created index.html and style.css files for the HTML and CSS codes respectively.
- Downloaded images from Figma for my project design.
- Added linters file to check errors in the code.
- Added a descriptive README.md file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link href='https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="main-header">
+    <nav class="header-nav">
+      <a href="#" class="nav-logo">Elijah</a>
```
Comment (RolandM99) [inline index.html:14]: - [ ] Please, Kindly remove the `a` tag for the logo inside the `nav` tag. the logo should be in the header but not in the navigation section.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          <li>
+            <img src="assets/images/linkedin.svg" alt="Linkedin">
+          </li>
+          <li>
+            <img src="assets/images/Medium.svg" alt="Medium">
+          </li>
+          <li>
+            <img src="assets/images/github.svg" alt="Github">
+          </li>
+          <li>
```
Comment (RolandM99) [inline index.html:41]: - [ ] Good job so far by implementing the list of social media icons, but you should add a link for each icon. below is an example of how to implement them:
```
<ul>
    <li><a href="your_link_here"><img src="..." alt="..."></a></li>
    <li><a href="your_link_here"><img src="..." alt="..."></a></li>
</ul>
```

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @Elijahdre , 

Good job so far! you are almost there!
But, there are some simple issues that you still need to work on to go to the next project but your project looks good.


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @Elijahdre , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**